articles/what-is-ransomware/index.md

```diff
@@ -0,0 +1,50 @@
+<!--more-->
+### What is Ransomware?
+But before panicking, the first thing you can do to protect yourself is be informed. There are several kinds of cyber attacks to look out for, and one of them is Ransomware.
+
+Ransomware is a type of malware––or malicious software. It is the kind of threat where a hacker blocks their victim from accessing their own device, unless a ransom is paid (which usually must be in virtual currency, such as bitcoin). The most common way ransomware infects a computer is via phishing emails containing harmful attachments or drive-by downloads. When infected, a device usually displays a message with an official logo such as the FBI or Interpol’s, implying that the victim’s device has been taken over by the authorities due to “illegal activity”. Attackers effectively scare their victims into believing false accusations of criminal activity, which then promptly leads to them paying the ransom and getting the attack over with.
+
+### Who does it target?
+When ransomware attacks first began, their biggest targets were individual systems (aka regular people). However, as hackers became more organized and successful with time, they began attacking small and mid-sized businesses more. The top three countries where such attacks occur are the United Kingdom, the United States, and Canada, presumably because of their large market sizes. Nevertheless, regular people still remain potential targets of ransomware attacks.
+
+### How does it operate?
+Ransomware uses something called cryptoviral extortion to operate, which is a three step process:
+- Attackers generate a pair of cryptographic keys––a public and a private––and place the public key in the malware. The private key remains solely with the attacker.
+- When released onto a device, the malware generates a random symmetric key to encrypt the victim’s data. The public key is then used to encrypt the symmetric key. This process is called hybrid encryption, and creates an asymmetric ciphertext of the victim’s data. It also erases the symmetric key and the victim’s original plaintext data. The only way a victim can recover their data is by paying the ransom and sending the asymmetric ciphertext to the attacker––which is part of the message the user sees.
+- Once the attacker receives their payment and the asymmetric ciphertext, they decrypt the data using their private key, thus successfully completing the attack.
+
+Ransomware is the type of malware that conceals its true intent––by masking itself as a “warning from the authorities.” In computing, such type of malware is called Trojan, since it misleads users by appearing unsuspicious.
+
+### What should you do if infected?
+Here are some things you should do if you believe your device is infected:
```

Comment:
"things you should do" => guidelines you should follow

Prior to this, there should be a way to figure out signs that your device may be infected.